### PR TITLE
Added a functionality to sort dropdown values manualy

### PR DESCRIPTION
**Pull request by romanblanco**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1594301

screencast from 2018-06-26 16-36-28

### CODE REVIEW

**Comment by romanblanco** [comment]
@miq-bot add_reviewer @Hyperkid123

**Comment by romanblanco** [comment]
@miq-bot add_label blocker

File: src/dialog-editor/components/modal-field-template/drop-down-list.html [hunk 1]
```diff
@@ -63,22 +63,46 @@
...
-               ng-blur="vm.entriesChange()"
-               name="label"
-               ng-model="entry[1]"
-               type="text"
-               placeholder="Value"
-               required/>
-        <a ng-click='vm.removeEntry(entry); vm.entriesChange()'><i class='pficon-close'></i></a>
+      <div ng-if='vm.modalData.options.sort_by !== "none"'>
+        <div ng-repeat="entry in vm.modalData.values">
+          <input id="label"
```
Comment (Hyperkid123) [inline src/dialog-editor/components/modal-field-template/drop-down-list.html]: Change id to unique value please.

File: src/dialog-editor/components/modal-field-template/drop-down-list.html [hunk 2]
```diff
@@ -63,22 +63,46 @@
...
+                 placeholder="Value"
+                 required/>
+          <a ng-click='vm.removeEntry(entry); vm.entriesChange()'><i class='pficon-close'></i></a>
+        </div>
+      </div>
+      <div ng-if='vm.modalData.options.sort_by === "none"'>
+        <div ui-sortable='vm.sortableOptionsValues'
+             ng-model='vm.modalData.values'>
+          <div class="draggable-field" ng-repeat="entry in vm.modalData.values">
+            <input id="label"
```
Comment (Hyperkid123) [inline src/dialog-editor/components/modal-field-template/drop-down-list.html]: Same here change id please.

**Comment by Hyperkid123** [comment]
@romanblanco change the `col-sm-5` to `col-sm-6` that should fix the input position.

**Comment by dclarizio** [comment]
@romanblanco looks good . . . as discussed, please do a follow up PR with any tests you can come up with.

**Comment by ZitaNemeckova** [comment]
@miq-bot remove_label gaprindashvili/yes

**Comment by himdel** [comment]
Released in 1.0.31

**Comment by himdel** [comment]
This is missing a master PR, possibly one that includes #311 and the fix for https://github.com/ManageIQ/ui-components/pull/310#discussion_r198458848 as well.

**Comment by romanblanco** [comment]
Correct. As is said in https://github.com/ManageIQ/ui-components/pull/311#issue-197698718 :

> I'll add a new PR with the changes done in #310 together with some [specs](https://github.com/ManageIQ/ui-components/pull/310#issuecomment-400359683) to `master`.

**Pull request merged**